.github/workflows/release.yml

```diff
@@ -1,151 +1,157 @@
-name: Release Charts
```

Comment:
Looks like something wrong the formatting. Hard to review.

Reply:
Fixed. Please take a look again.